packages/type-storage/src/storage.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright 2017-2018 @polkadot/storage authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+import { Storage } from './types';
+import * as wellKnownKeys from './wellKnownKeys';
+
+export const storage: Storage = {
+  wellKnownKeys // Prefill storage with wellKnownKeys, as not returned by state_getMetadata
```

Comment:
Naming - I think it will probably be better to call these something like `substrate` or `runtime` (since system is taken).

Reply:
Runtime created confusion with Metadata.ts types (also called Runtime*), so I preferred substrate.

Another idea is to have `storage.code`, `storage.authorityCount` etc, since they're not really in a module.